src/nrncvode/occvode.cpp

```diff
@@ -207,13 +208,17 @@ printf("%d Cvode::init_eqn id=%d neq_v_=%d #nonvint=%d #nonvint_extra=%d nvsize=
             // sentinal values for determining no_cap
             NODERHS(z.v_node_[i]) = 1.;
         }
-        for (i = 0; i < zneq_cap_v; ++i) {
-            auto* const node = z.cmlcap_->ml.size() == 1 ? z.cmlcap_->ml[0].nodelist[i]
-                                                         : z.cmlcap_->ml[i].nodelist[0];
-            z.pv_[i] = node->v_handle();
-            z.pvdot_[i] = node->rhs_handle();
-            *z.pvdot_[i] = 0.;  // only ones = 1 are no_cap
-        }
+        i = 0;
+        if (zneq_cap_v)
+            for (auto& ml: z.cmlcap_->ml) {
+                for (int j = 0; j < ml.nodecount; ++j) {
+                    auto* const node = ml.nodelist[j];
+                    z.pv_[i] = node->v_handle();
+                    z.pvdot_[i] = node->rhs_handle();
+                    *z.pvdot_[i] = 0.;  // only ones = 1 are no_cap
+                    ++i;
+                }
```

Comment:
```suggestion
                for (auto* node: ml.nodelist) {
                    z.pv_[i] = node->v_handle();
                    z.pvdot_[i] = node->rhs_handle();
                    *z.pvdot_[i] = 0.0;  // only ones = 1 are no_cap
                    ++i;
                }
```

should be functionally equivalent.

Reply:
Is that possible without changing the ml structure (perhaps a good idea for the future). Presently ```ml.nodelist``` comes from nrnoc_ml.h
```
struct Memb_list {
    ...
    Node** nodelist{};
```

(Note that with regard to changing Memb_list, there is a comment:
```
 * Because this type is passed from NEURON library code into code generated from MOD files, it is
 * prone to ABI issues -- particularly when dealing with Python wheels.
```
)

Reply:
Ah, so it's not a container, but a raw pointer? Then yeah, my suggestion is not applicable.